node/src/types/block.rs

```diff
@@ -536,7 +554,8 @@ impl FinalizedBlock {
             );
         }
         let random_bit = rng.gen();
-        let block_payload = BlockPayload::new(deploys, vec![], vec![], random_bit);
+        let block_payload =
+            BlockPayload::new(deploys, vec![], vec![], Default::default(), random_bit);
```

Comment:
Similar here - it would be good to create some random value for `PastFinalitySignatures`.

Reply:
Done